packages/content-conformance/src/rules.ts

```diff
@@ -128,6 +145,9 @@ export async function loadRule(
     // local rule - relative not found
   }
 
+  // ensure the rule severity level gets included so we can reference it when reporting
+  rule.level = level
```

Comment:
This `level` assignment looks like it is only needed to patch the `internal rule` case right?
- `convertRemarkLintRule()` returns the level properly.

NIT: Maybe move this alongside the internalRule assignment

```diff
  rule = internalRule
+ rule.level = level
```

Reply:
There are other cases that need this assignment: internal rules and local rules. They are loaded below the remark lint rule `try/catch` block. I can likely refactor this to make it more clear though 👀